core/src/util/yamlHelper.cpp

```diff
@@ -64,30 +64,44 @@ glm::vec4 getColorAsVec4(const Node& node) {
 }
 
 std::string parseSequence(const Node& node) {
+    if (node.IsSequence()) { return ""; }
+
     std::stringstream sstream;
     for (const auto& val : node) {
-        try {
-            sstream << val.as<double>() << ",";
-        } catch (const BadConversion& e) {
-            try {
-                sstream << val.as<std::string>() << ",";
-            } catch (const BadConversion& e) {
-                LOGE("Float or Unit expected for styleParam sequence value");
-            }
+        if (!val.IsScalar()) {
+            // LOG
+            return "";
+        }
+
+        double value;
+        if (getDouble(val, value)) {
```

Comment:
Ok this is something required. Seems to fix the issue I pointed out in this PR. Any color which is a sequence of doubles, get bad values and hence messes up the colors. Also visible in the default scene file.

![screenshot from 2016-12-05 20-57-49](https://cloud.githubusercontent.com/assets/360641/20910096/80591d2e-bb2d-11e6-8a78-859364d6297d.png)